schema/me/lot_standing.js

```diff
@@ -22,6 +25,13 @@ export const LotStandingType = new GraphQLObjectType({
     bidder: {
       type: Bidder.type,
     },
+    is_in_live_auction: {
+      type: GraphQLBoolean,
+      description: "This lot is associated with a live auction",
+      resolve: ({ bidder }) => {
+        return gravity(`sale/${bidder.sale.id}`).then(sale => isLiveAuction(sale))
```

Comment:
Use the `saleLoader` instead https://github.com/artsy/metaphysics/blob/99ea872fd3f5f12f330a0d90a366a4d84fa913d8/lib/loaders/loaders_without_authentication/index.js#L23